components/theme/Container.tsx

```diff
@@ -5,11 +5,32 @@
  * @param {node} props.children - The child elements of the container.
  * @returns {JSX.Element} A section element with the appropriate styles applied.
  */
-const Container = ({ ...props }) => {
-  const { children } = props;
+import clsx from "clsx";
 
+interface ContainerProps {
+  maxW?:
+    | "sm"
+    | "md"
+    | "lg"
+    | "xl"
+    | "2xl"
+    | "3xl"
+    | "4xl"
+    | "5xl"
+    | "screen"
+    | "screen-xl";
+  children: React.ReactNode;
+  className?: string;
+}
+const Container = ({
+  maxW = "screen-xl",
+  children,
+  className,
+}: ContainerProps) => {
   return (
-    <section className="mx-auto max-w-screen-xl px-5 xl:px-16">
+    <section
+      className={clsx("mx-auto px-5 xl:px-16", 'max-w-' + maxW, className)}
```

Comment:
Tailwind in the past, with purgecss didnot like these string interpolated classNames, I am not sure, regardless lets have full classNames instead. We could define a conditional here instead.

This is what I did on Badge component
https://github.com/envless/envless/blob/04ef6cf9fb641ddfa487ae3ee12837a7c3763413/components/theme/Badge.tsx#L12-L17

Reply:
Thanks.. this is also done 🥳